app/src/main/java/com/duckduckgo/app/global/api/AtpPixelRemovalInterceptor.kt

```diff
@@ -54,7 +54,47 @@ class AtpPixelRemovalInterceptor @Inject constructor() : Interceptor, PixelInter
         private const val PIXEL_PREFIX = "m_atp_"
 
         // list here the pixels that except from this interceptor
-        private val PIXEL_EXCEPTIONS = listOf<String>()
+        private val PIXEL_EXCEPTIONS = listOf(
+            "m_atp_ev_enabled_tracker_activity_d",
+            "m_atp_ev_enabled_tracker_activity_c",
+            "m_atp_ev_enabled_tracker_activity_u",
+            "m_atp_imp_exclusion_list_activity_d",
+            "m_atp_imp_exclusion_list_activity_c",
+            "m_atp_imp_exclusion_list_activity_u",
+            "m_atp_imp_company_trackers_activity_d",
+            "m_atp_imp_company_trackers_activity_c",
+            "m_atp_imp_company_trackers_activity_u",
+            "m_atp_imp_tracker_activity_detail_d",
+            "m_atp_imp_tracker_activity_detail_c",
+            "m_atp_imp_tracker_activity_detail_u",
+            "m_atp_ev_selected_cancel_app_protection_c",
+            "m_atp_ev_selected_disable_app_protection_c",
+            "m_atp_ev_selected_disable_protection_c",
+            "m_atp_ev_disabled_tracker_activity_c",
+            "m_atp_ev_enabled_on_search_d",
+            "m_atp_ev_disabled_on_search_d",
+            "m_atp_ev_enabled_on_launch_d",
+            "m_atp_ev_enabled_on_launch_c",
+            "m_atp_ev_disabled_on_launch_d",
+            "m_atp_ev_disabled_on_launch_c",
+            "m_atp_ev_enabled_on_search_c",
+            "m_atp_ev_disabled_on_search_c",
+            "m_atp_ev_enabled_onboarding_u",
```

Comment:
The `m_atp_ev_enabled_onboarding_u` along with all pixels down to `m_atp_imp_disable_app_protection_detail_c` are subject to https://app.asana.com/0/69071770703008/1204382677817470/f